### PR TITLE
Update pl-PL.txt - improved translation, added some missing strings

### PR DESCRIPTION
**Pull request by Maq4all**
improved translation, added some missing strings

### CODE REVIEW

File: data/language/pl-PL.txt [hunk 1]
```diff
@@ -563,6 +563,97 @@ STR_1316    :Graczy
...
 STR_1319    :Gracz {COMMA16}
+STR_1320    :kurs
+STR_1321    :kursy
+STR_1322    :Kurs
+STR_1323    :Kursy
+STR_1324    :{COMMA16} kurs
+STR_1325    :{COMMA16} kursy
+STR_1326    :Kurs {COMMA16}
+STR_1327    :{SMALLFONT}{BLACK}Obróć obiekt o 90{DEGREE}
+STR_1328    :Level land required
```
Comment (Nubbie) [inline data/language/pl-PL.txt]: English string, forgotten to be translated

File: data/language/pl-PL.txt [hunk 2]
```diff
@@ -563,6 +563,97 @@ STR_1316    :Graczy
...
+STR_1355    :{WINDOW_COLOUR_2}Spadki: {BLACK}{COMMA16}
+STR_1356    :{WINDOW_COLOUR_2}Inversions: {BLACK}{COMMA16}
+STR_1357    :{WINDOW_COLOUR_2}Holes: {BLACK}{COMMA16}
+STR_1358    :{WINDOW_COLOUR_2}Całkowity czas 'w powietrzu': {BLACK}{COMMA2DP32}secs
+STR_1359    :{WINDOW_COLOUR_2}Czas w kolejce: {BLACK}{COMMA16} minut
+STR_1360    :{WINDOW_COLOUR_2}Czas w kolejce: {BLACK}{COMMA16} minut
+STR_1361    :Nie można zmienić prędkości...
+STR_1362    :Nie można zmienić prędkości startowej...
+STR_1363    :Za wysoko na wsporniki!
+STR_1364    :Supports for track above can't be extended any further!
```
Comment (Nubbie) [inline data/language/pl-PL.txt]: English string, forgotten to be translated

File: data/language/pl-PL.txt [hunk 3]
```diff
@@ -563,6 +563,97 @@ STR_1316    :Graczy
...
+STR_1376    :Lower Transfer
+STR_1377    :Heartline Roll (left)
+STR_1378    :Heartline Roll (right)
+STR_1379    :Reverser (left)
+STR_1380    :Reverser (right)
+STR_1381    :Curved Lift Hill (left)
+STR_1382    :Curved Lift Hill (right)
+STR_1383    :Quarter Loop
+STR_1384    :{YELLOW}{STRINGID}
+STR_1385    :{SMALLFONT}{BLACK}Other track configurations
```
Comment (Nubbie) [inline data/language/pl-PL.txt]: English string, forgotten to be translated + all above

File: data/language/pl-PL.txt [hunk 4]
```diff
@@ -563,6 +563,97 @@ STR_1316    :Graczy
...
+STR_1401    :Wyjście
+STR_1402    :{SMALLFONT}{BLACK}Wybuduj lub przesuń wejście do atrakcji
+STR_1403    :{SMALLFONT}{BLACK}Wybuduj lub przesuń wyjście z atrakcji 
+STR_1404    :{SMALLFONT}{BLACK}Obrót o{DEGREE}
+STR_1405    :{SMALLFONT}{BLACK}Zdjęcie lustrzane
+STR_1406    :{SMALLFONT}{BLACK}Włącz/wyłącz scenerię (jeśli dostępne w motywie)
+STR_1407    :{WINDOW_COLOUR_2}Wybuduj to...
+STR_1408    :{WINDOW_COLOUR_2}Koszt: {BLACK}{CURRENCY}
+STR_1409    :Platforma wejścia/wyjścia
+STR_1410    :Vertical Tower
```
Comment (Nubbie) [inline data/language/pl-PL.txt]: English string, forgotten to be translated

File: data/language/pl-PL.txt [hunk 5]
```diff
@@ -2198,6 +2289,12 @@ STR_3314    :Wprowadź nazwę dla scenariusza:
 STR_3315    :Szczegóły parku/scenariusza
 STR_3316    :Wprowadź opis tego scenariusza:
 STR_3317    :Brak szczegółów
+STR_3318    :{SMALLFONT}{BLACK}Wybierz w jakich grupach występuje ten scenariusz
+STR_3319    :{WINDOW_COLOUR_2}Grupa scenariuszy:
+STR_3320    :Nie można zapisać pliku scenariusza...
+STR_3321    :Nowe elementy zainstalowano pomyślnie
+STR_3322    :{WINDOW_COLOUR_2}Cel: {BLACK}{STRINGID}
+STR_3323    :Missing object data, ID:
```
Comment (Nubbie) [inline data/language/pl-PL.txt]: English string, forgotten to be translated

File: data/language/pl-PL.txt [hunk 6]
```diff
@@ -2294,8 +2391,21 @@ STR_5123    :Odnów atrakcje
 STR_5126    :Losowa muzyka tytułowa
 STR_5128    :Rozmiar zaznaczenia
 STR_5130    :Rozmiar mapy
-STR_5132    :Napraw przejażdżki
+STR_5132    :Napraw wszystkie atrakcje
+STR_5133    :{SMALLFONT}{BLACK}Adjust smaller area of land rights
+STR_5134    :{SMALLFONT}{BLACK}Adjust larger area of land rights
```
Comment (Nubbie) [inline data/language/pl-PL.txt]: English string, forgotten to be translated

**Comment by Nubbie** [comment]
As the language is so out of date, it's a good thing it got updated 👍 
Maybe you would want to be a maintainer for it?

**Comment by Gymnasiast** [comment]
Hi, I see that a previous contributor put American scenario names in the translation file, rather than the original British ones. Could you correct these names?
|American name|Original name|
|---------------|--------------|
|Mothball Mountain|Mystic Mountain|
|Dinky Park|Pokey Park|
|Big Pier|Paradise Pier|
|Big Pier 2|Paradise Pier 2|
|Aqua Park|White Water Park|
|Haunted Harbor|Haunted Harbour|
|Katie's World|Katie's Dreamland|

**Comment by Maq4all** [comment]
Sorry, i dont have a enough time to be a full-time maintainer for this language. Now i out of office for few days (vacation), so i have some free time to do few updates.

File: data/language/pl-PL.txt [hunk 7]
```diff
@@ -635,22 +863,22 @@ STR_1478    :{WINDOW_COLOUR_2}Ocena mdłości: {BLACK}{COMMA2DP32}  ({STRINGID})
...
 STR_1490    :{SMALLFONT}{OPENQUOTES}{STRINGID} ma naprawdę dobrą cenę{ENDQUOTES}
 STR_1491    :{SMALLFONT}{OPENQUOTES}Mam już {STRINGID}{ENDQUOTES}
 STR_1492    :{SMALLFONT}{OPENQUOTES}Nie mogę sobie pozwolić na {STRINGID}{ENDQUOTES}
 STR_1493    :{SMALLFONT}{OPENQUOTES}Nie jestem głodny{ENDQUOTES}
-STR_1494    :{SMALLFONT}{OPENQUOTES}Nie jestem spragniony{Eć NDQUOTES}
+STR_1494    :{SMALLFONT}{OPENQUOTES}Nie jestem spragniony{ENDQUOTES}
 STR_1495    :{SMALLFONT}{OPENQUOTES}Pomocy! Tonę!{ENDQUOTES}
 STR_1496    :{SMALLFONT}{OPENQUOTES}Zgubiłem się!{ENDQUOTES}
-STR_1497    :{SMALLFONT}{OPENQUOTES}{STRINGID} było świetne{ENDQUOTES}
+STR_1497    :{SMALLFONT}{OPENQUOTES}Na atrakcji {STRINGID} było świetne{ENDQUOTES}
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: świetn**i**e

File: data/language/pl-PL.txt [hunk 8]
```diff
@@ -1759,6 +2013,16 @@ STR_2678    :???
 STR_2679    :???
 # END OF CONTROLS LIST
 STR_2681    :{MEDIUMFONT}{BLACK}Zwiększa twoją gotówkę o {CURRENCY}
+STR_2684    :{SMALLFONT}{BLACK}Przybywające duże grupy gości
+STR_2685    :Parametr hałasu Simplex
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: w tym kontekście nie "hałas", ale "szum".

File: data/language/pl-PL.txt [hunk 9]
```diff
@@ -2041,6 +2305,50 @@ STR_2962    :{WINDOW_COLOUR_2}
...
+STR_2969    :<removed string - do not use>
+STR_2970    :<removed string - do not use>
+STR_2971    :Podstawowy schemat koloru
+STR_2972    :Alternatywny schemat koloru nr 1
+STR_2973    :Alternatywny schemat koloru nr 2
+STR_2974    :Alternatywny schemat koloru nr 3
+STR_2975    :{SMALLFONT}{BLACK}Wybierz schemat kolorów malowania atrakcji
+STR_2976    :{SMALLFONT}{BLACK}Pomaluj elementy atrakcji korzystająć z indywidualnego schematu koloru
+STR_2977    :Nazwa pracownika
+STR_2978    :Wprowadż nazwę tego pracownika:
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: ź

File: data/language/pl-PL.txt [hunk 10]
```diff
@@ -2041,6 +2305,50 @@ STR_2962    :{WINDOW_COLOUR_2}
...
+STR_2979    :Nie można zmienić nazwy pracownika...
+STR_2980    :Za dużo banerów w grze
+STR_2981    :{RED}Brak wejścia - - 
+STR_2982    :Tekst baneru
+STR_2983    :Wprowadż nową treść baneru:
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: ź

File: data/language/pl-PL.txt [hunk 11]
```diff
@@ -2041,6 +2305,50 @@ STR_2962    :{WINDOW_COLOUR_2}
...
+STR_2984    :Nie można wprowadzić nowego treść baneru...
+STR_2985    :Baner
+STR_2986    :{SMALLFONT}{BLACK}Zmień treść baneru
+STR_2987    :{SMALLFONT}{BLACK}Ustaw ten baner jako{NEWLINE}'zakaz wstępu' dla gości
+STR_2988    :{SMALLFONT}{BLACK}Usuń ten baner
+STR_2989    :{SMALLFONT}{BLACK}Wybierz główny kolor
+STR_2990    :{SMALLFONT}{BLACK}Wybierz kolor treści
+STR_2991    :Znak
+STR_2992    :Treść znaku
+STR_2993    :Wprowadż nową nazwę znaku:
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: ź

File: data/language/pl-PL.txt [hunk 12]
```diff
@@ -2105,7 +2413,7 @@ STR_3113    :Wybierz inny projekt
 STR_3114    :{SMALLFONT}{BLACK}Wróć do okna wyboru projektu
 STR_3115    :{SMALLFONT}{BLACK}Zapisz projekt trasy
 STR_3116    :{SMALLFONT}{BLACK}Zapisz projekt trasy (niedostępne do czasu sprawdzenia atrakcji i wygenerowania statystyk)
-STR_3117    :{BLACK}Wzywane jest mechanik...
+STR_3117    :{BLACK}Wzywane jest mechanik...ć
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: ?

File: data/language/pl-PL.txt [hunk 13]
```diff
@@ -2120,11 +2428,64 @@ STR_3128    :Zapisz Projekt Trasy
 STR_3129    :Zapisz Projekt Trasy ze Scenerią
 STR_3130    :Zapisz
 STR_3131    :Anuluj
+STR_3132    :{BLACK}Wybierz elementy scenerii który chcesz zapisać z porjektem trasy...
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: któr**e**; p**ro**jektem

File: data/language/pl-PL.txt [hunk 14]
```diff
@@ -2120,11 +2428,64 @@ STR_3128    :Zapisz Projekt Trasy
...
+STR_3133    :Nie można tego zbudować na zboczu
+STR_3134    :{RED}(Projekt zawiera niedostępne elementy scenerii)
+STR_3135    :{RED}(Projekt wagoników niedostępny - może mieć to wpływ na atrakcyjność atrakcji)
+STR_3136    :Uwaga: Ten projekt trasy będzie wybudowany z alternatywnymi wagonami, dlatego może nie działać zgodnie z oczekiwaniami
+STR_3137    :Wybierz najbliższą scenerie
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: ę

File: data/language/pl-PL.txt [hunk 15]
```diff
@@ -2120,11 +2428,64 @@ STR_3128    :Zapisz Projekt Trasy
...
 STR_3157    :mapa
 STR_3158    :wykres
 STR_3159    :lista
+STR_3160    :{SMALLFONT}{BLACK}Wybierz liczbę obrotów na przejażdzkę
+STR_3161    :<removed string - do not use>
+STR_3162    :Nie można zaalokować odpowiedniej ilości pamięci
+STR_3163    :Instalowanie nowych danych: 
+STR_3164    :{BLACK}{COMMA16} wybrano (maksymalnie {COMMA16})
+STR_3165    :
+STR_3166    :{BLACK}(ID:
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: Isn't it missing some placeholder here?

File: data/language/pl-PL.txt [hunk 16]
```diff
@@ -2120,11 +2428,64 @@ STR_3128    :Zapisz Projekt Trasy
...
+STR_3177    :Nie można odznaczyć tego obiektu
+STR_3178    :Przynajmniej jeden obiekt ścieżki musi zostać wybrany
+STR_3179    :Przynajmniej jedna atrakcja musi zostać wybrana
+STR_3180    :Nieprawidłowy wybór obiektu
+STR_3181    :Wybór obiektu - {STRINGID}
+STR_3182    :Wejście do parku musi zostać wybrane
+STR_3183    :Rodzaj wody musi zostać wybrany
+STR_3184    :Atrakcje przejazdowe/pojazdy
+STR_3185    :Drobna scenografia
+STR_3186    :Duża scenogrania
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: scenogra**f**ia

File: data/language/pl-PL.txt [hunk 17]
```diff
@@ -2198,7 +2559,29 @@ STR_3314    :Wprowadź nazwę dla scenariusza:
 STR_3315    :Szczegóły parku/scenariusza
 STR_3316    :Wprowadź opis tego scenariusza:
 STR_3317    :Brak szczegółów
+STR_3318    :{SMALLFONT}{BLACK}Wybierz w jakich grupach występuje ten scenariusz
+STR_3319    :{WINDOW_COLOUR_2}Grupa scenariuszy:
+STR_3320    :Nie można zapisać pliku scenariusza...
+STR_3321    :Nowe elementy zainstalowano pomyślnie
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: bardziej naturalne wydaje mi się "zainstalowan**e**", podmiotem zdają się być "nowe elementy".

File: data/language/pl-PL.txt [hunk 18]
```diff
@@ -2198,7 +2559,29 @@ STR_3314    :Wprowadź nazwę dla scenariusza:
...
+STR_3322    :{WINDOW_COLOUR_2}Cel: {BLACK}{STRINGID}
+STR_3323    :Missing object data, ID:
 STR_3324    :Wymaga dodatku: {STRINGID}
+STR_3325    :Wymaga zawartości dodatkowej
+STR_3326    :{WINDOW_COLOUR_2}(brak obrazka)
+STR_3327    :Nieustawiona pozycja poczatkowa dla ludzi
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: ą

File: data/language/pl-PL.txt [hunk 19]
```diff
@@ -2198,7 +2559,29 @@ STR_3314    :Wprowadź nazwę dla scenariusza:
...
+STR_3328    :Nie można przejść do dalszego etapu edytora...
+STR_3329    :Nie wybudowano jeszcze wejścia do parku
+STR_3330    :Park musi posiadać jakiś teren na własność
+STR_3331    :Path from park entrance to map edge either not complete or too complex - Path must be single-width with as few junctions and corners as possible
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: Should these be added as verbatim to translated files? I thought these should be left blank to have the engine fallback onto the default en-GB. @Gymnasiast ?

File: data/language/pl-PL.txt [hunk 20]
```diff
@@ -2305,9 +2703,9 @@ STR_5151    :.
 #Decimals separator
 STR_5152    :,
 STR_5153    :Edytuj motywy...
-STR_5154    :Hardware display
+STR_5154    :Grafika sprzętowa
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: To nie jest poprawne tłumaczenie. Powinno być: "Wyświetlanie sprzętowe"

File: data/language/pl-PL.txt [hunk 21]
```diff
@@ -2375,18 +2775,62 @@ STR_5236    :Okno:
...
+STR_5273    :{SMALLFONT}{BLACK}Duża Sceneria
+STR_5274    :{SMALLFONT}{BLACK}Ścieżka
+STR_5275    :Szukaj Obiektu
+STR_5276    :Podaj nazwę obiektu który chcesz znaleźć
+STR_5277    :Wyczyść
 STR_5278    :Tryb piaskownicy
 STR_5279    :Tryb piaskownicy wyłączony
 STR_5285    :BUM!!!
 STR_5286    :{SMALLFONT}{BLACK}Niektórzy goście eksplodują
+STR_5287    :Atrakcja jet zepsuta
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: je**s**t

File: data/language/pl-PL.txt [hunk 22]
```diff
@@ -2375,18 +2775,62 @@ STR_5236    :Okno:
...
 STR_5288    :Atrakcja jest zamknięta
+STR_5289    :Brak uszkodzeń dla tej atrakcji
 STR_5290    :Napraw przejażdżkę
+STR_5291    :Nie można wymusić awarii
+STR_5292    :{SMALLFONT}{BLACK}Wymuś awarię
+STR_5293    :{SMALLFONT}{BLACK}Zamknij atrakcję
+STR_5294    :{SMALLFONT}{BLACK}Testuj atrakcję
+STR_5295    :{SMALLFONT}{BLACK}Otwórz atrakcję
+STR_5296    :{SMALLFONT}{BLACK}Zamknij park
+STR_5297    :{SMALLFONT}{BLACK}OTwórz Park
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: O**t**wórz

File: data/language/pl-PL.txt [hunk 23]
```diff
@@ -2375,18 +2775,62 @@ STR_5236    :Okno:
...
+STR_5298    :{RED}{STRINGID}
+STR_5299    :{LIGHTPINK}{STRINGID}
+STR_5300    :{SMALLFONT}{BLACK}Szybkie zwolnienie pracownika
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: To chyba się tyczy trybu, gdzie można zwolnić więcej niż jednego pracownika -> "Szybkie zwalnianie pracowników"

File: data/language/pl-PL.txt [hunk 24]
```diff
@@ -2439,19 +2906,29 @@ STR_5432    :Komenda:
 STR_5435    :Zmień nazwę zapisu
 STR_5436    :Edytuj sekwencje tytułowe...
 STR_5437    :Nie wybrano zapisu
+STR_5438    :Nie można dokonać zmian dopóki edytor komand jest uruchomiony
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: kom**e**nd

File: data/language/pl-PL.txt [hunk 25]
```diff
@@ -2439,19 +2906,29 @@ STR_5432    :Komenda:
...
+STR_5439    :Przynajmniej 4 sekundowa komenda oczekiwania jest wymagana przy użyciu komendy restartu
+STR_5440    :Zmniej utratę ostrości przy pełnym ekranie
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: zmniej**sz**?

File: data/language/pl-PL.txt [hunk 26]
```diff
@@ -2511,7 +2998,7 @@ STR_5561    :Nie udało się załadować języka
 STR_5562    :UWAGA!
 STR_5563    :Ta funkcja jest obecnie niestabilna, zachowaj szczególną ostrożność.
 STR_5564    :Dodaj uszkodzone elementy
-STR_5565    :{SMALLFONT}{BLACK}Inserts a corrupt map element at top of tile. This will hide any element above the corrupt element.
+STR_5565    :{SMALLFONT}{BLACK}Wstaw uszkodzony element w górnej części dachówki. Spowoduje to ukrycie elementów znajdujących się nad uszkodzony elementem.
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: nie "dachówki", raczej "kafelka"

File: data/language/pl-PL.txt [hunk 27]
```diff
@@ -2615,19 +3105,19 @@ STR_5724    :Burza
 STR_5725    :{BLACK}Wymuś pogodę:
 STR_5726    :{SMALLFONT}{BLACK}Ustawia aktualną pogodę w parku
 STR_5727    :Jakość skalowania
-STR_5728    :Requires hardware display option
-STR_5729    :{SMALLFONT}{BLACK}Requires hardware display option
+STR_5728    :Wymaga włączoną grafikę sprzętową
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: wyświetlanie sprzętowe, także poniżej

File: data/language/pl-PL.txt [hunk 28]
```diff
@@ -2678,286 +3168,286 @@ STR_5786    :Nieprawidłowa nazwa grupy
...
+STR_5791    :{SMALLFONT}{BLACK}Ustaw niezawodność wszystkich atrakcji na 100%{NEWLINE}i zresetuj datę budowy na "w tym roku"
+STR_5792    :{SMALLFONT}{BLACK}Naprawia wszystkie wszystkie uszkodzone atrakcje
+STR_5793    :{SMALLFONT}{BLACK}Usuwa historię katastrof,{NEWLINE}dzięki czemu goście nie będą narzekać że jest niebezpieczne
+STR_5794    :{SMALLFONT}{BLACK}Niektóre scenariusze blokują edycję{NEWLINE}istniejących juz w parku atrakcji.{NEWLINE}Ta opcja obchodzi te restrykcje
+STR_5795    :{SMALLFONT}{BLACK}Goście korzystają ze wszystkich atrakcji w parku{NEWLINE}nawet jeśli ich intensywność jest ekstremalna
+STR_5796    :{SMALLFONT}{BLACK}Wymusza zamknięcie/otwarcie parku
+STR_5797    :{SMALLFONT}{BLACK}Wyłącza zmienność pogody{NEWLINE}i ustawia wybrany jej rodzaj
+STR_5798    :{SMALLFONT}{BLACK}Pozwala budować atrakcje w trakcie pauzy
+STR_5799    :{SMALLFONT}{BLACK}Wyłącza awarie i katastrofy związane z uszkodzeniem hamulców
+STR_5800    :{SMALLFONT}{BLACK}Zapobiera awarii atrakcji
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: Zapobie**g**a

File: data/language/pl-PL.txt [hunk 29]
```diff
@@ -2678,286 +3168,286 @@ STR_5786    :Nieprawidłowa nazwa grupy
...
-STR_5818    :{SMALLFONT}{BLACK}[Requires hardware display]{NEWLINE}Use nearest neighbour scaling{NEWLINE}when window scaling factor set{NEWLINE}to integer values (1, 2, 3, etc)
-STR_5819    :{SMALLFONT}{BLACK}[Requires hardware display]{NEWLINE}Pause the game if Steam{NEWLINE}in-game overlay is opened
-STR_5820    :{SMALLFONT}{BLACK}Minimise the game if focus is{NEWLINE}lost while in fullscreen mode
-STR_5821    :{SMALLFONT}{BLACK}Changes the colour of the construction marker when building rides,{NEWLINE}paths, shops, scenery, etc.
-STR_5822    :{SMALLFONT}{BLACK}Cycle between day and night.{NEWLINE}Full cycle takes one in-game month
-STR_5823    :{SMALLFONT}{BLACK}Display banners in uppercase (RCT1 behaviour)
-STR_5824    :{SMALLFONT}{BLACK}Disables lightning effect{NEWLINE}during a thunderstorm
-STR_5825    :{SMALLFONT}{BLACK}Keep the mouse cursor in the window
+STR_5816    :{SMALLFONT}{BLACK}Ustawia stosunek skalowania grafiki.{NEWLINE}Przydatne szczególnie przy grze{NEWLINE}w wysokiej rozdzielczości
+STR_5817    :{SMALLFONT}{BLACK}[Wymaga sterownika sprzętowego]{NEWLINE}Ustaw tryb skalowania UI
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: wyświetlania sprzętowego, podobnie poniżej

File: data/language/pl-PL.txt [hunk 30]
```diff
@@ -2678,286 +3168,286 @@ STR_5786    :Nieprawidłowa nazwa grupy
...
+STR_5847    :{SMALLFONT}{BLACK}Wybierz sekwencje parków na ekran tytułowy.{NEWLINE}Sekwencje tytułowe z RCT1/2 wymagaja zaimportowanych odpowiednich scenariuszy
+STR_5848    :{SMALLFONT}{BLACK}Twórz i zarządzaj sekwencjami tytułowymi
+STR_5849    :{SMALLFONT}{BLACK}Automatycznie rozlokuj{NEWLINE}nowo zatrudniony personel
+STR_5850    :{SMALLFONT}{BLACK}Nowo zatrudniony dozorca domyślnie {NEWLINE}kosi trawę (zaszłość RCT1)
+STR_5851    :{SMALLFONT}{BLACK}Ustaw domyślną częstotliwość inspekcji{NEWLINE}na nowych atrakcjach
+STR_5852    :{SMALLFONT}{BLACK}Ustaw nazwę kanału TwitchTV który będzie używany przy integracji z Twitch
+STR_5853    :{SMALLFONT}{BLACK}Przełącznik włączania/wyłączania efektów dźwiękowych
+STR_5854    :{SMALLFONT}{BLACK}Przełącznik włączania/wyłączania dźwięków na atrakcjach
+STR_5855    :{SMALLFONT}{BLACK}Ustaw normalny pełny ekran, bezramkowy pełny ekran{NEWLINE}klub tryb okienkowy
+STR_5856    :{SMALLFONT}{BLACK}Ustaw rozdzielność ekranu w trybie pełnoekranowym
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: rozdziel**cz**ość

File: data/language/pl-PL.txt [hunk 31]
```diff
@@ -2678,286 +3168,286 @@ STR_5786    :Nieprawidłowa nazwa grupy
...
 STR_5857    :{SMALLFONT}{BLACK}Ustawienia gry
-STR_5858    :{SMALLFONT}{BLACK}Use GPU for displaying instead of CPU. Improves compatibility with screen capture software. May slightly decrease performance.
-STR_5859    :{SMALLFONT}{BLACK}Enables frame tweening for visually{NEWLINE}smoother gameplay. When disabled,{NEWLINE}the game will run at 40 FPS.
-STR_5860    :Toggle original/decompiled track drawing
+STR_5858    :{SMALLFONT}{BLACK}Użyć do wyświetlania GPU zamiast CPU. Pomoże to w poprawie kompatybilności z oprogramowaniem do przechwytywania ekranu. Jednak może wpłynąć negatywnie na wydajność.
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: może "Użyj" albo "Używa"?

File: data/language/pl-PL.txt [hunk 32]
```diff
@@ -2678,286 +3168,286 @@ STR_5786    :Nieprawidłowa nazwa grupy
...
+STR_5859    :{SMALLFONT}{BLACK}Włącz zmienną ilość klatek dla {NEWLINE}płynniejszej gry. Gdy wyłączone,{NEWLINE}gra będzie używała stałych 40 FPS.
+STR_5860    :Przełącznik original/decompiled rysowania trasy
 STR_5861    :Błąd weryfikacji klucza
 STR_5862    :Blokuj nieznanych graczy.
-STR_5863    :{SMALLFONT}{BLACK}Only allow players with known keys to join.
+STR_5863    :{SMALLFONT}{BLACK}Pozwól na połączenie graczy znających klucz.
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: Pozwól na połączenie tylko graczy ze znanym kluczem.

File: data/language/pl-PL.txt [hunk 33]
```diff
@@ -2678,286 +3168,286 @@ STR_5786    :Nieprawidłowa nazwa grupy
...
 STR_5864    :Dostęp do serwera mają tylko gracze z białej listy.
 STR_5865    :Logowanie histori czatu
-STR_5866    :{SMALLFONT}{BLACK}Logs all chat history to files in your user directory.
-STR_5867    :{WINDOW_COLOUR_2}Provider Name: {BLACK}{STRING}
-STR_5868    :{WINDOW_COLOUR_2}Provider E-mail: {BLACK}{STRING}
-STR_5869    :{WINDOW_COLOUR_2}Provider Website: {BLACK}{STRING}
+STR_5866    :{SMALLFONT}{BLACK}Zapisój treść czatów do plików w katalogu domowym.
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: Zapis**u**j

File: data/language/pl-PL.txt [hunk 34]
```diff
@@ -2678,286 +3168,286 @@ STR_5786    :Nieprawidłowa nazwa grupy
...
+STR_5867    :{WINDOW_COLOUR_2}Nazwa hostującego: {BLACK}{STRING}
+STR_5868    :{WINDOW_COLOUR_2}E-mail hostującego: {BLACK}{STRING}
+STR_5869    :{WINDOW_COLOUR_2}Strona hostującego: {BLACK}{STRING}
```
Comment (janisozaur) [inline data/language/pl-PL.txt]: I end review here for now – will resume it tomorrow.

**Comment by janisozaur** [comment]
Overall the translation quality is very good, but proofreading it showed some minor mistakes.

I haven't reviewed it all yet due to lack of time, I'll try resuming it tomorrow.

File: data/language/pl-PL.txt (hunk 19 above)
Comment (Gymnasiast) [inline data/language/pl-PL.txt]: Indeed.

Comment (Gymnasiast) [inline data/language/pl-PL.txt]: Indeed.

**Comment by Gymnasiast** [comment]
@Maq4all Could you make the changes that @janisozaur requested?

**Comment by AaronVanGeffen** [comment]
The original author hasn't been active on GitHub since late July… @janisozaur, would you be able to wrap this up?

**Comment by Omaranwa** [comment]
lt looks dead :(

File: data/language/pl-PL.txt [hunk 35]
```diff
@@ -387,6 +427,103 @@ STR_1050    :Nie udało się wczytać...{NEWLINE}Plik zawiera błędne dane!
...
+STR_1067    :Upward launch
+STR_1068    :Rotating lift mode
+STR_1069    :Tryb od stacji do stacji
+STR_1070    :Pojedyńczy przejazd na wstęp
+STR_1071    :Nieograniczone przejażdy za wstęp
+STR_1072    :Tryb labiryntu
+STR_1073    :Tryb wyścigowy
+STR_1074    :Tryb zbijaków
+STR_1075    :Swing mode
+STR_1076    :Shop stall mode
```
Comment (Broxzier) [inline data/language/pl-PL.txt]: This line and the one above are both English strings.

File: data/language/pl-PL.txt [hunk 36]
```diff
@@ -387,6 +427,103 @@ STR_1050    :Nie udało się wczytać...{NEWLINE}Plik zawiera błędne dane!
...
+STR_1077    :Tryb obrotowy
+STR_1078    :Obroty do przodu
+STR_1079    :Obroty do tyłu
+STR_1080    :Film: {ENDQUOTES}Avenging aviators{ENDQUOTES}
+STR_1081    :3D film: {ENDQUOTES}Mouse tails{ENDQUOTES}
```
Comment (Broxzier) [inline data/language/pl-PL.txt]: This line and the one above are both English strings.

File: data/language/pl-PL.txt [hunk 37]
```diff
@@ -387,6 +427,103 @@ STR_1050    :Nie udało się wczytać...{NEWLINE}Plik zawiera błędne dane!
...
+STR_1082    :Tryb gwiezdnych pierścieni
+STR_1083    :Tryb początkującegoca
+STR_1084    :LIM-powered launch
+STR_1085    :Film: {ENDQUOTES}Thrill riders{ENDQUOTES}
+STR_1086    :3D film: {ENDQUOTES}Storm chasers{ENDQUOTES}
+STR_1087    :3D film: {ENDQUOTES}Space raiders{ENDQUOTES}
+STR_1088    :Intense mode
+STR_1089    :Berserk mode
```
Comment (Broxzier) [inline data/language/pl-PL.txt]: This line and the 5 lines above are all English strings.

File: data/language/pl-PL.txt [hunk 38]
```diff
@@ -387,6 +427,103 @@ STR_1050    :Nie udało się wczytać...{NEWLINE}Plik zawiera błędne dane!
...
+STR_1090    :Tryb nawiedzonego domu
+STR_1091    :Tryb cyrkowy
+STR_1092    :Downward launch
+STR_1093    :Crooked house mode
+STR_1094    :Tryb wolnego spaku
+STR_1095    :Sekcyjny tryb pracy ciągłej
+STR_1096    :Powered launch (without passing station)
+STR_1097    :Powered launch block sectioned mode
```
Comment (Broxzier) [inline data/language/pl-PL.txt]: This line and the line above are both English strings.

File: data/language/pl-PL.txt [hunk 39]
```diff
@@ -2198,7 +2559,29 @@ STR_3314    :Wprowadź nazwę dla scenariusza:
...
+STR_3323    :Missing object data, ID:
 STR_3324    :Wymaga dodatku: {STRINGID}
+STR_3325    :Wymaga zawartości dodatkowej
+STR_3326    :{WINDOW_COLOUR_2}(brak obrazka)
+STR_3327    :Nieustawiona pozycja poczatkowa dla ludzi
+STR_3328    :Nie można przejść do dalszego etapu edytora...
+STR_3329    :Nie wybudowano jeszcze wejścia do parku
+STR_3330    :Park musi posiadać jakiś teren na własność
+STR_3331    :Path from park entrance to map edge either not complete or too complex - Path must be single-width with as few junctions and corners as possible
+STR_3332    :Park entrance is the wrong way round or has no path leading to the map edge
```
Comment (Broxzier) [inline data/language/pl-PL.txt]: This line and the line above are both English strings.

File: data/language/pl-PL.txt [hunk 40]
```diff
@@ -2198,7 +2559,29 @@ STR_3314    :Wprowadź nazwę dla scenariusza:
...
+STR_3333    :Eksportuj elementy dodatkowe do zapisów gier
+STR_3334    :{SMALLFONT}{BLACK}Select whether to save any additional plug-in object data required (add-in data not supplied with the main product) in saved game or scenario files, allowing them to be loaded by someone who doesn't have the additional object data
```
Comment (Broxzier) [inline data/language/pl-PL.txt]: English string.

File: data/language/pl-PL.txt [hunk 41]
```diff
@@ -3009,19 +3499,250 @@ STR_6115    :Powered giant 4 x 4 trucks which can climb steep slopes
...
+[BNOODLES]
+STR_NAME    :Stoisko makaronu z wołowiną
+STR_DESC    :Stoisko na którym sprzedawany jest makaron z wołowiną
+
+[CHCKS]
+STR_NAME    :Budka z kurczakiem
+STR_DESC    :Budka w której sprzedawany jest smażony kurczak.
+
+[HOTDS]
+STR_NAME    :Hotdog
```
Comment (Broxzier) [inline data/language/pl-PL.txt]: English string. _(Possibly the same, I don't know Polish)_

File: data/language/pl-PL.txt (hunk 15 above)
Comment (Gymnasiast) [reply data/language/pl-PL.txt]: No, all languages have this.

**Pull request merged**